docs/developer-docs/getting-started/tokens-and-cycles.mdx

```diff
@@ -0,0 +1,147 @@
+
+There are three primary ways to use ICP tokens:
+
+- Convert ICP into [cycles](#cycles), which are required to pay for a <GlossaryTooltip>canister's</GlossaryTooltip> resources. **All dapp developers must acquire cycles to deploy their canister onto the mainnet.**
+
+- Trade ICP for other assets, such as other tokens.
+
+- Stake ICP in a [neuron](/docs/current/developer-docs/daos/nns/concepts/neurons/neuron-overview) in return for network governance participation and rewards.
+
+## Cycles
+
+The Internet Computer runs on a network of nodes owned and operated by a community of independent node providers distributed across the globe. Node providers spend money to run and maintain nodes, such that they purchase hardware and pay for the electricity and network bandwidth used by the node. The Internet Computer compensates these node providers on a monthly basis by minting and distributing rewards in the form of ICP tokens.
+
+Canisters consume the resources provided by these nodes when they are deployed on the mainnet. To make ICP sustainable, canisters are required to pay for the resources they use.
+
+Resource consumption is not paid in ICP tokens but **cycles**. As the canister is used, its cycles balance is continuously reduced. Eventually, the canister needs to be "topped up" with more cycles. The canister’s developer is responsible for topping up the canister.
+
+To get cycles, you must convert ICP tokens into cycles.
+
+:::info Why are cycles different from ICP tokens?
+
+While the value of an ICP token is volatile, cycles are not. Cycles are pegged to [XDR](https://en.wikipedia.org/wiki/Special_drawing_rights), a group of fiat currencies. This allows resource consumption and canister execution on ICP to be a rather stable price.
+:::
+
+## Other token types
+
+The Internet Computer supports creating and deploying custom fungible or non-fungible tokens through the [ICRC standards](/docs/current/developer-docs/defi/tokens/token-standards). Each ICRC standard defines a set of parameters that must be met to adhere to the standard. Not all ICRC standards are for tokens; some are for notifications, architectures, or other network functions.
+
+## What is a ledger?
+
+A [**ledger**](/docs/current/developer-docs/defi/tokens/ledger/overview) is a system canister used to store accounts and their transactions. Each developer identity has a **ledger account identifier**.
+
+Each token on the Internet Computer has its **own** ledger. This means that there are separate ledgers for ICP tokens, cycles, and every ICRC token that has been deployed on the network.
+
+[Learn more about ledgers](/docs/current/developer-docs/defi/tokens/ledger/overview).
+
+## Obtaining cycles
+
+You need to obtain ICP tokens, then convert them into cycles using `dfx`. However, you may also get a small number of free cycles from the [cycles faucet](https://anv4y-qiaaa-aaaal-qaqxq-cai.ic0.app/).
+
+### 1. Get your account's ledger account ID.
```

Comment:
Could you explain the `account-id` more? I remember being confused with the account id vs. principal early on